adcm_aio_client/core/objects/cm.py

```diff
@@ -333,6 +334,12 @@ async def cluster(self: Self) -> Cluster | None:
     async def hostprovider(self: Self) -> HostProvider:
         return await HostProvider.with_id(requester=self._requester, object_id=self._data["hostprovider"]["id"])
 
+    @async_cached_property
+    async def maintenance_mode(self: Self) -> MaintenanceMode:
+        response = await self._requester.get(*self.get_own_path())
+        self._data = response.as_dict()
```

Comment:
don't reread on cached property request

there's a `refresh` mechanism for all of that

MM should be the one retrieved on host retrieval from API

User will refresh object when they want "current" value

---

It's also a good idea to add `WithMaintenanceMode` mixin

Reply:
added

Reply:
no, it's not what I've meant

User should call refresh when they feel adequate, not we calling it or any similar code in here

pls use existing data for that, no requests

Reply:
changed